datadog-dashboards-dashboard-handler/datadog-dashboards-dashboard.json

```diff
@@ -0,0 +1,79 @@
+{
+  "typeName": "Datadog::Dashboards::Dashboard",
+  "description": "Datadog Dashboard",
```

Comment:
I'm also trying to standardize adding the version number of the resource in this field (see PR #97) If that gets approved, can we add the version here